File: java/project/moida/moida-jpa/src/main/java/com/brainbackdoor/moida/domain/content/Item.java
package com.brainbackdoor.moida.domain.content;


import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class Item {
    @Id
    @GeneratedValue
    private Long id;

    private String content;

    private boolean confirm;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public boolean isConfirm() {
        return confirm;
    }

    public void setConfirm(boolean confirm) {
        this.confirm = confirm;
    }
}

File: java/project/moida/moida-kotlin/src/main/kotlin/com/brainbackdoor/moidakotlin/domain/content/Item.kt
package com.brainbackdoor.moidakotlin.domain.content

import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.Id

@Entity
data class Item(
        val content: String,

        val confirm: Boolean,

        @Id
        @GeneratedValue
        val id: Long = 0
)